src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -577,8 +641,114 @@ const MeasureGroups = () => {
             )}
             {activeTab === "stratification" && (
               <FormControl>
-                <Divider />
-                <FieldLabel>Stratification support to come</FieldLabel>
+                {formik.values.stratifications.map((strat, i) => (
+                  <Row>
```

Comment:
Fine for now but feels like this whole row can be a separate component?

Reply:
Added a comment